pkg/agent/interfacestore/types.go

```diff
@@ -134,3 +135,7 @@ func NewUplinkInterface(uplinkName string) *InterfaceConfig {
 	uplinkConfig := &InterfaceConfig{InterfaceName: uplinkName, Type: UplinkInterface}
 	return uplinkConfig
 }
+
+func (c *InterfaceConfig) GetIPv4Addr() net.IP {
```

Comment:
Question - I understand you want not to change all features right now. However, for the longer term, it might be be good to keep most code use net.IP, but not v4 or v6 addresses?

Reply:
Correct, I plan to change features one by one, so this function will be removed after all feature have been modified to support dual-stack. I will add a "Todo" here to remind to remove this function.